vocabularies/Common.json

```diff
@@ -1288,6 +1288,51 @@
       "@Common.Experimental": true,
       "@Core.IsURL": true,
       "@Core.Description": "Base URL for WebSocket connections"
+    },
+    "RecommendationsAction": {
+      "$Kind": "Term",
+      "$AppliesTo": ["EntityType"],
```

Comment:
See below: ... So the targets are relative to the entity set for which the action is provided. Only that the action is annotated at the entity type which might be used for several entity sets (in theory).

Reply:
Annotation now targets the entity set.

Reply:
still says $AppliesTo: EntityType

Reply:
It now targets the entity type again, because the entity set can be derived from the binding parameter.

Reply:
> the entity set can be derived from the binding parameter

As long as we only have one entity set per type.